### PR TITLE
Feature localization

### PR DESCRIPTION
**Pull request by AmirSoln**

### CODE REVIEW

File: assets/translations/en-US.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "SETTINGS_PAGE": "Settings",
+  "LOADING": "Loading...",
+  "NO_RESULTS": "No results",
+  "SERVER_CONNECTION_ERROR": "Server connection could not be established!",
+  "WELCOME": "Welcome!",
+  "ENTER_SERVER_INFO": "Please enter your Ombi server address,",
+  "INFO_STORED_WARNING": "The information supplied will only be stored locally.",
+  "SERVER_ADDRESS": "Server Address",
+  "NEXT_BUTTON": "Next",
+  "__SETTINGS_PAGE_SECTION__": "Settings page related translations",
```
Comment (tomerblecher) [inline assets/translations/en-US.json]: Remove
```suggestion
```

File: assets/translations/en-US.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "SEARCH_DELAY": "Search Delay",
+  "SEARCH_DELAY_INFO": "Delay should be between 1-5 seconds",
+  "DEFAULT_CONTENT": "Default Content",
+  "DEFAULT_CONTENT_INFO": "Default content to load on search",
+  "RESET_DEFAULTS": "Reset Default",
+  "RESET_DEFAULTS_INFO": "Double tap to reset settings",
+  "RESET_BUTTON": "Reset",
+  "RESET_BUTTON_CONFIRM": "Confirm",
+  "RESET_SUCCESSFUL": "The Settings reset completed successfully!",
+  "__SNACK_BAR_SECTION__": "Snack bar related translations",
```
Comment (tomerblecher) [inline assets/translations/en-US.json]: Remove
```suggestion
```

File: assets/translations/he.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "SETTINGS_PAGE": "הגדרות",
+  "LOADING": "אנא המתן...",
+  "NO_RESULTS": "לא נמצאו תוצאות",
+  "SERVER_CONNECTION_ERROR": "ארעה שגיאה בעת ניסון התחברות לשרת!",
+  "WELCOME": "ברוכים הבאים!",
+  "ENTER_SERVER_INFO": "אנא הכנס את כתובת שרת הOmbi שלך,",
+  "INFO_STORED_WARNING": "המידע שנרשם יישמר אך ורק מקומית על המכשיר.",
+  "SERVER_ADDRESS": "הכנס כתובת שרת",
+  "NEXT_BUTTON": "שלב הבא",
+  "__SETTINGS_PAGE_SECTION__": "Settings page related translations",
```
Comment (tomerblecher) [inline assets/translations/he.json]: Remove
```suggestion
```

File: assets/translations/he.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  "SEARCH_DELAY": "דיליי חיפוש",
+  "SEARCH_DELAY_INFO": "הדיליי צריך להיות בין 1 ל-5 שניות",
+  "DEFAULT_CONTENT": "תוכן ברירת מחדל",
+  "DEFAULT_CONTENT_INFO": "תוכן ברירת המחדל שייטען בעמוד החיפוש",
+  "RESET_DEFAULTS": "איפוס לברירת מחדש",
+  "RESET_DEFAULTS_INFO": "יש ללחוץ לחיצה כפולה על הכפתור על מנת לאפס",
+  "RESET_BUTTON": "אפס",
+  "RESET_BUTTON_CONFIRM": "אשר",
+  "RESET_SUCCESSFUL": "האיפוס להגדרות ברירת המחדל הסתיים בהצלחה!",
+  "__SNACK_BAR_SECTION__": "Snack bar related translations",
```
Comment (tomerblecher) [inline assets/translations/he.json]:
```suggestion
```

File: lib/main.dart [hunk 5]
```diff
@@ -46,5 +50,12 @@ Future<void> main() async {
   await secureStorage.init();
   await localSettings.init();
 
-  runApp(OcneraApp());
+  // runApp(OcneraApp());
+  runApp(EasyLocalization(
+      supportedLocales: [Locale('he', 'IL')],
```
Comment (tomerblecher) [inline lib/main.dart]:
```suggestion
      supportedLocales: [Locale('he', 'IL'), Locale('en', 'US')],
```

File: lib/main.dart [hunk 6]
```diff
@@ -46,5 +50,12 @@ Future<void> main() async {
...
+      path: 'assets/translations',
+      startLocale: Locale('he', ''),
```
Comment (tomerblecher) [inline lib/main.dart]:
```suggestion
```

File: lib/pages/login/server_config.dart [hunk 7]
```diff
@@ -49,14 +50,13 @@ class _ServerConfigState extends State<ServerConfig> {
                   children: <Widget>[
                     Column(
                       children: <Widget>[
-                        Text("Welcome!",
+                        Text('WELCOME'.tr(),
                             style:
                                 TextStyle(color: Colors.orange, fontSize: 40)),
-                        Text("Please enter your Ombi server address,",
+                        Text('ENTER_SERVER_INFO'.tr(),
```
Comment (tomerblecher) [inline lib/pages/login/server_config.dart]: ENTER_SERVER_ADDRESS

File: lib/pages/root.dart [hunk 8]
```diff
@@ -50,7 +51,7 @@ class _RootPageState extends State<RootPage> {
                   height: 50,
                 ),
                 Text(
-                  "Loading...",
+                  "LOADING".tr(),
```
Comment (tomerblecher) [inline lib/pages/root.dart]:
```suggestion
                  'LOADING'.tr(),
```

File: lib/pages/search/top_bar.dart [hunk 9]
```diff
@@ -54,7 +55,7 @@ class _TopBarState extends State<TopBar> {
             controller: _editingController,
             cursorColor: Colors.orange,
             decoration: InputDecoration(
-              hintText: "Search..",
+              hintText: 'SEARCH_TOP_BAR'.tr(),
```
Comment (tomerblecher) [inline lib/pages/search/top_bar.dart]: SEARCH_INPUT_PLACEHOLDER

**Review by tomerblecher — CHANGES_REQUESTED** [review]

File: lib/pages/login/server_config.dart (hunk 7 above)
Comment (AmirSoln) [reply lib/pages/login/server_config.dart]: https://github.com/STRDCoders/Ocnera/pull/39/commits/0bb2842a55ba9d9152f1809e2d967b1ea905be80

File: lib/pages/search/top_bar.dart (hunk 9 above)
Comment (AmirSoln) [reply lib/pages/search/top_bar.dart]: https://github.com/STRDCoders/Ocnera/pull/39/commits/d8e4b327e9ad6cc2681ebe808ce44082b90d67f8

**Review by tomerblecher — APPROVED** [review]

**Pull request merged**